Accessible Deadzones (#438)
Leafwing-Studios/leafwing-input-manager: RELEASES.md

- Added `InputMap::Clear`.
- Fixed [a bug](https://github.com/Leafwing-Studios/leafwing-input-manager/issues/430) related to incorrect axis data in `Chord` when not all buttons are pressed.

### Enhancements

> - Changed `DualAxis` and `SingleAxis` deadzones to be more accessible. 

Reviewer: "More accessible" is really unclear to the reader.

Author: That's true. I could remove the that bullet point and just list the points underneath? I hoping that the following bullets would explain it, but I understand how that can be unclear.

Reviewer: ```suggestion
- Improved deadzone handling for both `DualAxis` and `SingleAxis` deadzones 
```

  - All deadzones now scale the input so that it is continuous.
  - `DeadZoneShape::Cross` handles each axis seperately, making a per-axis "snapping" effect.

>   - An input on the line of a deadzone is now considering inside.

Reviewer: ```suggestion
  - An input on the boundary of a deadzone is now considering inside.

```
